src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.cs

```diff
@@ -1278,6 +1279,76 @@ public void RegisterMissingIncludes(BlittableJsonReaderArray results, BlittableJ
             }
         }
 
+        internal void RegisterCounters(BlittableJsonReaderArray counters, bool gotAll, string[] ids)
```

Comment:
what are the `ids` here for, documents ids, or counter names?

Reply:
Document ids